Dockerfile

```diff
@@ -21,7 +21,13 @@ COPY .npmrc .
 RUN pnpm install --store=pnpm-store
 
 # copy the rest of the content
-COPY . /usr/app
+COPY src /usr/app/src
+COPY typings /usr/app/typings
+COPY nuxt-template-overrides /usr/app/nuxt-template-overrides
+COPY tsconfig.json /usr/app/tsconfig.json
+COPY nuxt.config.ts /usr/app/nuxt.config.ts
+COPY tailwind.config.js /usr/app/tailwind.config.js
+COPY tailwind.safelist.txt /usr/app/tailwind.safelist.txt
```

Comment:
Would it be easier to specify a `.dockerignore` file with explicit allow statements and just run `COPY . /usr/app`? That's what we do [in the API](https://github.com/WordPress/openverse-api/blob/main/api/.dockerignore)

Reply:
Probably!